README.md

```diff
@@ -573,7 +573,6 @@ chatSession.onAutoDisconnection(event => {
 ```js
 chatSession.onConnectionLost(event => {
   const { chatDetails, data } = event;
-  switch (data.ContentType) {
     // ...
   }
```

Comment:
Nitpick: I think this should be removed.

Reply:
What do you mean by _this_?  The `const { chatDetails, data } = event;` line or the entire README block on `chatSession.onConnectionLost()`?

Reply:
Also the `// ...` probably need to be reindented.